pandas/core/series.py

```diff
@@ -299,8 +306,12 @@ def __init__(
         if fastpath:
 
             # data is an ndarray, index is defined
-            if not isinstance(data, SingleBlockManager):
-                data = SingleBlockManager.from_array(data, index)
+            if not isinstance(data, SingleManager):
+                manager = get_option("mode.data_manager")
+                if manager == "block":
+                    data = SingleBlockManager.from_array(data, index)
+                elif manager == "array":
+                    data = SingleArrayManager.from_array(data, index)
```

Comment:
This is something I have below another time, so could move it to a helper function to deduplicate